lib/linguist/heuristics.rb

```diff
@@ -150,8 +150,17 @@ def call(data)
       end
     end
 
-    disambiguate "AsciiDoc", "AGS Script" do |data|
-      Language["AsciiDoc"] if /^=+(\s|\n)/.match(data)
+    disambiguate "AsciiDoc", "AGS Script", "Public Key" do |data|
+      if /^[=-]+(\s|\n)|{{[A-Za-z]/.match(data)
+        Language["AsciiDoc"]
+      elsif /^(\/\/.+|((import|export)\s+)?(function|int|float|char)\s+((room|repeatedly|on|game)_)?([A-Za-z]+[A-Za-z_0-9]+)\s*[;\(])/.match(data)
+        Language["AGS Script"]
+      elsif /^-----BEGIN/.match(data)
+        Language["Public Key"]
+      else
+        # Plain text can also be AsciiDoc.
+        Language["AsciiDoc"]
```

Comment:
@bkeepers - what do you think about having this as the fallback. Given the relative use of `"AsciiDoc", "AGS Script", "Public Key"` it seems like a reasonable assumption. That said, perhaps we should just return nothing and let the classifier do its thing.